frontend/src/stores/user.js

```diff
@@ -93,11 +89,28 @@ export const useUserStore = defineStore("user", {
     },
 
     /**
-     * Set the current authenticated user.
-     * @param {object} user - User object.
+     * Set the current user based on authenticated user's ID.
+     */
+    setCurrentUser() {
+      const authStore = useAuthStore(); // Usar el store de auth
```

Comment:
Change comment to english

Reply:
Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/5/commits/75f045424c57e6fbc9fb053661f5d23294a22f09)